src/index.css

```diff
@@ -1,5 +1,6 @@
 body {
   margin: 0;
+ /* background-color: #610a15;*/
```

Comment:
Not sure why this was added if it's just commented out anyway :/

Reply:
Actually i was testing it to make it look in different color.